components/gitpod-protocol/src/util/tracing.ts

```diff
@@ -28,7 +28,8 @@ export namespace TraceContext {
         if (!!parentCtx?.span) {
             const ctx = parentCtx?.span?.context();
             if (ctx && !!ctx.toTraceId() && !!ctx.toSpanId()) {
-                options.references = [opentracing.followsFrom(ctx)];
+                // options.references = [opentracing.followsFrom(ctx)];
+                options.childOf = ctx;
```

Comment:
@geropl, wdyt, should we remove these entirely? 
frankly, I don't understand the issue at hand, but given I started https://github.com/gitpod-io/gitpod/pull/9281, I'd like to find an end to this story. 
what option do we have? if we revert, we end up with messed up logs.

Reply:
@geropl, is this still an issue?

Reply:
No, I think we should leave it as-is. The problems we faced were mostly solved by Mad's attempts to find dangling spans that were not `finish()`ed. :+1: